release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -274,6 +274,17 @@ The `openshiftVersion` parameter of the `AgentServiceConfig` custom resource (CR
 
 For more information see, xref:../scalability_and_performance/routing-optimization.adoc#configuring-ingress-controller-router-liveness-readiness-and-startup-probes[Configuring Ingress Controller (router) Liveness, Readiness, and Startup probes]
 
+[id=ocp-4-11-node-observability-operator]
+==== Node Observability Operator (Technology Preview)
+{product-title} 4.11 introduces the Node Observability Operator in Technology Preview.
+
+Node Observability Operator provides the following features:
+
+* Trigger a CRIO-O and Kubelet profiling.
```

Comment:
I'd put this second just for the logical sequence: we deploy the agent before we trigger the profiling.

Reply:
done. Updated.